pkgs/by-name/pu/pulsemeeter/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  libpulseaudio,
+  libappindicator,
+  gobject-introspection,
+  gtk3,
+
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "pulsemeeter";
+  version = "1.2.14";
+
+  src = fetchFromGitHub {
+    owner = "theRealCarneiro";
+    repo = "pulsemeeter";
+    rev = "v${version}";
```

```suggestion
    tag= "v${finalAttrs.version}";
```

`tag` is usually used over `rev` in order to not accidentally fetch a branch when looking for a fixed release